tests/ui/documents/test_documents_filter.py

```diff
@@ -138,10 +142,25 @@ def test_identifiedby_format():
     ]
     results = [
         {
-            'type': 'Isbn', 'value': '9782844267788'},
+            'details': '',
```

Comment:
What happens for `{% if identifier.details %}` if `details` is "".
Should it be better to have no details field if it is empty?

Reply:
If there is no details, nothing is displayed. That's perhaps not the best solution, but as far as tested it does what's expected. Do you have an improvement suggestion?